content/blog/2017-11-24-unconf-welcome.md

```diff
@@ -0,0 +1,70 @@
+---
+slug: "unconf-welcome"
+title: "The Value of Welcome, part 2: How to prepare 40 new community members for an unconference"
+date: 2017-11-24
+preface: "I’m participating in the AAAS Community Engagement Fellows Program, funded by the Alfred P. Sloan Foundation. The inaugural cohort of Fellows comprises 17 community managers working in a wide range of scientific communities. This is cross-posted from the Trellis blog as part of a series of reflections the Fellows are sharing."
+authors:
+  - name: Stefanie Butland
+categories: blog
+topicid:
+tags:
+- R
+- community
+- unconf
+- ropensci-team
+- welcome
+---
+
+I’ve raved about the value of [extending a personalized welcome](https://ropensci.org/blog/2017/07/18/value-of-welcome/) to new community members and I recently shared [six tips for running a successful hackathon-flavoured unconference](https://ropensci.org/blog/2017-11-17-unconf-sixtips). Building on these, I’d like to share the specific approach and (free!) tools I used to help prepare new rOpenSci community members to be productive at our [unconference](http://unconf17.ropensci.org/). My approach was inspired directly by my [AAAS Community Engagement Fellowship Program](https://blog.trelliscience.com/introducing-the-2017-community-engagement-fellows/) (AAAS-CEFP) training. Specifically, 1) one mentor said that the most successful conference they ever ran involved having one-to-one meetings with all participants prior to the event, and 2) prior to our in-person AAAS-CEFP training, we completed an intake questionnaire that forced us to consider things like “what do you hope to get out of this” and “what do you hope to contribute”.
```

Comment:
this link format `https://ropensci.org/blog/2017-11-17-unconf-sixtips` is wrong, date with slashes separating numbers, not dashes

Reply:
fixed